apps/ideaspace/src/components/common/IdeaCard/IdeaCardTag.js

```diff
@@ -1,10 +1,10 @@
 import React, { useState } from 'react';
-// import theme from '@devlaunchers/components/styles/theme';
 import { StatuBox } from './StyledIdeaCard';
 
 const IdeaCardTag = ({ status }) => {
-  const [tag, setTag] = useState('');
-  // const [tagColor, setTagColor] = useState('');
+  if (status == 'applying') {
+    status = 'in-review';
```

Comment:
@Hindu-Priya 
props are immuatable so you can't assign different values to it if I'm not mistaken.

Reply:
other than that this PR is approved by me

Reply:
@Enjoy2Live that may be true in TypeScript, but I don't believe the same applies to JavaScript. Source: https://www.reddit.com/r/reactjs/comments/14howx9/why_im_able_to_change_props_value_if_props_are/?rdt=59264

Reply:
it's not a typescript or javascript thing, it's a react thing.

in react you can't change props this way as props are immutable
> Don’t try to “change props”. When you need to respond to the user input (like changing the selected color), you will need to “set state”, which you can learn about in [State: A Component’s Memory.](https://react.dev/learn/state-a-components-memory)

Source: https://react.dev/learn/passing-props-to-a-component#how-props-change-over-time

Reply:
I made a new commit, ensuring props immutability in the component. Thanks!